### DISCUSSION TITLE
/mnt/docker-aio-config is not a mountpoint. Cannot proceed!

### DISCUSSION THREAD

**Discussion by Rhodderz** [nextcloud/all-in-one#2293] [Questions] [answered]
### Steps to reproduce
1. Update via Nextcloud-Master webui or new install

### Expected behavior

Nextcloud either installs or updates

### Actual behavior 

Nextcloud Master container spams 
/mnt/docker-aio-config is not a mountpoint. Cannot proceed
Also the mount has all the files missing suddenly as well.

### Host OS
Ubuntu 22.04.2 LTS running in LXC on Proxmox

#### Nextcloud AIO version
Interface Unavailable

#### Current channel
Latest and Beta

#### Other valuable info
Interestingly if i run this interactivly it "works" but as a whole new system as if it has no config and /mnt/docker-aio-config/data has no data in it and also does not look like it is mounting /var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data internally or even the mountpoint i specified

Running in interactive mode with the below command I am also able to get the mount info as per issue:2030

```
root@nextcloud:/var/lib/docker/volumes# docker run -it --name nextcloud-aio-mastercontainer --restart always -p 80:80 -p 8080:8080 -p 8443:8443 --volume nextcloud_aio_mastercontainer:/mnt/nc-aio/config --volume /var/run/docker.sock:/var/run/docker.sock:ro -e NEXTCLOUD_DATADIR="/mnt/nc-aio/data" nextcloud/all-in-one:20230112_091330-latest
Trying to fix docker.sock permissions internally...
Creating docker group internally with id 999
 Storage Driver: vfs
Warning: It seems like the storage driver vfs is used. This will lead to problems with disk space and performance and is disrecommended!
Generating a RSA private key
```

```
grep mastercontainer /proc/self/mountinfo
1936 1721 0:107 /var/lib/docker/volumes/nextcloud_aio_mastercontainer/_data /mnt/nc-aio/config rw,noatime master:191 - zfs bfspool/subvol-126-disk-0 rw,xattr,posixacl
```

This has been working previously as well with no changes to the host
I have since updated everything i can on the host and on the LXC instance incase that was the issue as well but to no luck
The LXC Container has also been given mount=nfs,nesting=1,fuse=1 which has worked fine since

I have also tried putting the below in my LXC config for this machine, updating the docker command to point the volume mounts to that instead and still getting the same error
mp0: /bfspool/share/nc-aio,mp=/mnt/nc-bind

**Comment by Zoey2936** [comment]
From where did you get this docker run command? Please use this: https://github.com/nextcloud/all-in-one#how-to-use-this

**Reply by Zoey2936** [reply, marked as answer]
@Rhodderz did you read this? Your start command it completely wrong.

**Reply by Rhodderz** [reply]
Right, yup somehow in me bouncing between work and this i for some reason changed the volume mount I apologise no excuse bar long day lol and thanks.
Fortunately Portainer kept the logs and going through the issue originally that caused the update to fail was 2 fold
1: me not updating for ages 
2: https://github.com/nextcloud/all-in-one/discussions/2065

Again thank you and apologies

**Comment by Rhodderz** [comment]
Apologies copied the wrong command from my terminal the 2 below are the ones I have ran, the -it one was when i was testing and trying to work out what was going on as it was stuck on restarting.

Also apologies for not starting this in discussions.

```
docker run \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
-p 80:80 \
-p 8080:8080 \
-p 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/nc-bind/config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
-e NEXTCLOUD_DATADIR="/mnt/nc-bind/data" \
nextcloud/all-in-one:latest
```

```
docker run \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
-p 80:80 \
-p 8080:8080 \
-p 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/nc-bind/config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
-e NEXTCLOUD_DATADIR="/mnt/nc-bind/data" \
nextcloud/all-in-one:beta
```

**Comment by Rhodderz** [comment]
Additionally, I have just created a brand new vm running Ubuntu 22.04.2 LTS
mounted the share via NFS 
And hit the exact same error

```
rhodderz@nextcloud:/mnt/nc-aio$ sudo docker run \
--sig-proxy=false \
--name nextcloud-aio-mastercontainer \
--restart always \
-p 80:80 \
-p 8080:8080 \
-p 8443:8443 \
--volume nextcloud_aio_mastercontainer:/mnt/nc-aio/config \
--volume /var/run/docker.sock:/var/run/docker.sock:ro \
-e NEXTCLOUD_DATADIR="/mnt/nc-aio/data" \
nextcloud/all-in-one:latest
Unable to find image 'nextcloud/all-in-one:latest' locally
latest: Pulling from nextcloud/all-in-one
63b65145d645: Pull complete
e80fdb865a69: Pull complete
f4dbeadc75c1: Pull complete
257b31b3391b: Pull complete
10633608be9b: Pull complete
467468bb17a8: Pull complete
bea1bb471b36: Pull complete
5928cc0dc6fd: Pull complete
da283a501353: Pull complete
82eedb1764b5: Pull complete
ab85c44693a8: Pull complete
a62153a6c4b2: Pull complete
4f4fb700ef54: Pull complete
d93d18265ee9: Pull complete
2e547c88daeb: Pull complete
8d306475680c: Pull complete
f44b7bfb1a20: Pull complete
6b30c4148a9a: Pull complete
7d2e92eadf60: Pull complete
400f9ea6d295: Pull complete
348e4106fa6e: Pull complete
3c5e6112e9eb: Pull complete
04dc452177da: Pull complete
7cf6abfa9fcf: Pull complete
530f8e6fad32: Pull complete
7e9f86b9532e: Pull complete
a05c4fd9aaf7: Pull complete
b9f7b9d6711c: Pull complete
08bb5bfea84f: Pull complete
0b07435461a1: Pull complete
ae53d2b41fd3: Pull complete
e2cf8cf4bd7f: Pull complete
Digest: sha256:73ee97576a8a1ad3f3331441df29b272d17c0fd420f26e56d00cdd8c704719cf
Status: Downloaded newer image for nextcloud/all-in-one:latest
/mnt/docker-aio-config is not a mountpoint. Cannot proceed!
```
